.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+name: iOS CI
+
+on: [push]
+
+jobs:
+  test:
+    name: "Build and test Velocidi SDK using [${{ matrix.os }}, xcode${{ matrix.xcode }}, sdk${{ matrix.sdk }}, ${{ matrix.device }}]"
+    env:
+      WORKSPACE: "VelocidiSDK.xcworkspace"
+      SCHEME: "VelocidiSDK"
+    strategy:
+      matrix:
+        include:
+         - os: "macos-10.15"
+           xcode: "10.3"
+           sdk: "12.4"
+           device: "iPhone X"
+         - os: "macos-10.15"
+           xcode: "11.7.0"
+           sdk: "13.7"
+           device: "iPhone 11"
+         - os: "macos-10.15"
```

Comment:
We would have to change the test structure a bit for that. Also, we probably would not gain that much with those tests, as we should not be relying on things that change between those versions. Given that, I would keep on testing only major versions.

Reply:
Given what we talked about in an offline conversation, there is really no benefit in us being so thorough in testing this against multiple SDK versions. Other iOS frameworks, like [AFNetworking](https://github.com/AFNetworking/AFNetworking/blob/master/.github/workflows/ci.yml#L29) don't really do that and they are far more complex and prone to use something that would cause errors in earlier versions than us. Testing in versions `12.4`, `13.7`, and `14.4` should be good enough